### DISCUSSION TITLE
Downloading file in Chinese file name ends up with garbage file name

### DISCUSSION THREAD

**Discussion by lokiahead** [subhra74/xdm#625] [Q&A] [answered]
Hi,
This is a very interesting program, and got to know it in Archlinux. I like it.
My question is, when I download a file with Chinese file name, and it ends up with a garbage file name after completion.
Any solution?
Thanks

**Comment by BraINstinct0** [comment]
does iconv help? I'm not sure, but it might be locale issues

**Comment by subhra74** [comment, marked as answer]
The issue will be fixed in next release

**Reply by lokiahead** [reply]
Appreciated!
